packages/wrangler/src/api/dev.ts

```diff
@@ -32,18 +32,31 @@ export async function unstable_dev(script: string, options: DevOptions) {
 		fetch: (init?: RequestInit) => Promise<Response | undefined>;
 	}>((resolve) => {
 		//lmao
+		let pidToKill: number | undefined;
```

Comment:
```suggestion
		let localDevServerProcessPID: number | undefined;
```
Let's be clear what this is, not what we do with it later  
Also, just to clarify, this won't exist if local: false, correct?

Reply:
curious, can this exist in the effect cleanup function inside `<Local/>`?